src/eli5.ts

```diff
@@ -0,0 +1,109 @@
+import { RequestBodyOpenAI, ResponseBodyOpenAi } from './types/index'
+
+/*
+This variable stores the request body as a JSON-formatted string for the "eli5" model. 
+It includes a system message to provide context for the conversation.
+*/
+const request_body_Eli5 = `{"model":"gpt-3.5-turbo","messages":[{"role":"system","content":"You are a Computer Science Teacher talking to his students and the students do not have a deep technical understanding. So the teacher tries to stay simple"},{"role":"user","content":""}]}`
```

Comment:
what does it mean that there are multiple messages ?

Reply:
that is just the format openAI expects. or what do you mean